.github/workflows/jdl.yml

```diff
@@ -88,14 +94,35 @@ jobs:
       - name: 'SETUP: get date'
         id: get-date
         run: echo "::set-output name=date::$(/bin/date -u "+%Y%m%d")"
-      - name: 'SETUP: docker'
+
+      - name: 'SETUP Linux: Set up swap space'
```

Comment:
Having SETUP and Set up twice looks kinda funny. Maybe SETUP should be changed to Configure?